pkg/storaged/jobs.js

```diff
@@ -132,6 +132,9 @@
                 'md-raid-fault-device':        _("Marking $target as faulty"),
                 'md-raid-remove-device':       _("Removing $target from RAID Device"),
                 'md-raid-create':              _("Creating RAID Device $target"),
+                'mdraid-check-job':            _("Checking RAID Device $target"),
```

Comment:
Where do we get these descriptions from? A comment [here](https://github.com/cockpit-project/cockpit/pull/4878/files#diff-1652bdcce9857918e7cf756ba1f40787L109) would be nice, especially since `mdraid` and `md-raid` is inconsistent.